docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/DefaultDockerCompose.java

```diff
@@ -200,7 +198,7 @@ public boolean writeLogs(String container, OutputStream output) throws IOExcepti
                     .until(() -> exists(container));
             Process executedProcess = followLogs(container);
             IOUtils.copy(executedProcess.getInputStream(), output);
-            executedProcess.waitFor(COMMAND_TIMEOUT.getMillis(), MILLISECONDS);
+            executedProcess.waitFor();
```

Comment:
Not sure why we had a timeout here. We probably shouldn't stop collecting logs after some arbitrary time if the container is still running. This process will be killed by the `shutdownNow` in `FileLogCollector#stopCollecting`.

Reply:
I think the rule of thumb was that every single `.waitFor` should have some kind of timeout to be 100% certain that tests will never hang forever

Reply:
Looking at this again, it's not clear to me why we need this `waitFor` at all. This is the last things we do in a executor thread so there's really no need to wait here.